docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1276,7 +1276,7 @@ the following to `rule.yml`:
 
     -   **arg_name** - Argument name, eg. `audit`
 
-    -   **arg_value** - Argument value, eg. `'1'`
+    -   **arg_value** - Argument value, eg. `'1'` (Optional)
```

Comment:
I feel like we should explain a bit more the template behavior when `arg_value` is not provided. For example, that the template will render a test that only looks for the presence of `arg_name`. The whole template is built around the idea of checking `argument=value`, so it might confuse people seeing the word `optional` with no further explanation. Sorry to nitpick here ;)

Reply:
OK, added more explanation on this doc, what do you think?